chia/_tests/util/test_replace_str_to_bytes.py

```diff
@@ -47,10 +47,10 @@
     MAX_GENERATOR_SIZE=uint32(1000000),
     MAX_GENERATOR_REF_LIST_SIZE=uint32(512),
     POOL_SUB_SLOT_ITERS=uint64(37600000000),
-    SOFT_FORK2_HEIGHT=uint32(0),
+    SOFT_FORK2_HEIGHT=uint32(0),  # unused
     SOFT_FORK4_HEIGHT=uint32(5650000),
     HARD_FORK_HEIGHT=uint32(5496000),
-    HARD_FORK_FIX_HEIGHT=uint32(5496000),
+    HARD_FORK_FIX_HEIGHT=uint32(0),  # unused
```

Comment:
Can these simply be removed from `ConsensusConstants` instead?  Or given default values perhaps?

Reply:
Planned to be done in conjunction with a chia_rs update.